extension-manifest-v3/src/_locales/en/messages.json

```diff
@@ -229,6 +261,12 @@
   "with_ghostery_enabled_you_instantly_browser_the_web_safer__faster_and_with_less_annoying_ads_": {
     "message": "With Ghostery enabled you instantly browser the web safer, faster and with less annoying ads."
   },
+  "yes": {
+    "message": "Yes"
+  },
+  "you_can_always_change_these_settings_in_the_nbsp__strong_ghostery_control_panel__strong__": {
+    "message": "You can always change these settings in the&nbsp;<strong>Ghostery control panel</strong>."
```

Comment:
wonder what the translator are going to think about this syntax. Can we make it simpler somehow?

Reply:
I am not sure what you mean by "simpler". `&nbsp;` is a normal word, and to avoid splitting the sentence into two keys, strong is inside of it (as the name can go somewhere else in other languages).